frontend-ssr/services/ExerciseSubmissions.service.ts

```diff
@@ -0,0 +1,18 @@
+import MOCK_SUBMISSIONS from '../pages/api/exercise-submissions/mock';
+
+const LOAD_DURATION = 1000;
+
+export default async function getSubmissions(page: number, query: string) {
+  let promise = new Promise((resolve, reject) => {
```

Comment:
You mixed `async/await` with Promises which is a big no-no. One or the other.

If you wanted to go with an async/await approach it would be something like:

```
async function getSubmissons(...){
  await wait(1000);
  return MOCK_SUBMISSIONS; 
}

function wait(mili) { 
  // Create another helper function which returns a promise that resolves after `mili` milliseconds
  return new Promise((resolve) => setTimeout(resolve, mili));
}


```